compact_str/src/tests.rs

```diff
@@ -503,14 +519,14 @@ fn test_extend_packed_from_empty() {
 #[test]
 fn test_pop_empty() {
     let num_pops = 256;
-    let mut compact = CompactString::from("");
-
-    (0..num_pops).for_each(|_| {
-        let ch = compact.pop();
-        assert!(ch.is_none());
-    });
-    assert!(compact.is_empty());
-    assert_eq!(compact, "");
+    for mut compact in [CompactString::from(""), CompactString::from_static_str("")] {
+        (0..num_pops).for_each(|_| {
+            let ch = compact.pop();
+            assert!(ch.is_none());
+        });
+        assert!(compact.is_empty());
+        assert_eq!(compact, "");
+    }
```

Comment:
nit: use the `#[test_case]` macro instead of a for loop here

Reply:
Thanks, I've applied your suggestion!